templates/admin/plugins/solr.tpl

```diff
@@ -28,6 +28,10 @@
 						<div class="form-group col-sm-6">
 							<label for="port">Port</label>
 							<input class="form-control" type="text" name="port" id="port" placeholder="8983" />
+						</div>
+							<div class="form-group col-sm-6">
```

Comment:
Sorry to be pedantic, but the indentation is wrong :smiley:

Reply:
@fixed